chromium_src/chrome/browser/ui/startup/startup_browser_creator.cc

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/startup/startup_browser_creator.h"
+#include "brave/browser/tor/tor_profile_manager.h"
+#include "brave/browser/tor/tor_profile_service_factory.h"
```

Comment:
this include is no longer used

Reply:
Fixed in https://github.com/brave/brave-core/pull/9533/commits/99e6e5a70b462e886bd65c8752374c7715d4386b.